lib/aws-efs-creator-stack.ts

```diff
@@ -4,6 +4,7 @@ import * as ec2 from 'aws-cdk-lib/aws-ec2';
 import * as efs from 'aws-cdk-lib/aws-efs';
 import * as iam from 'aws-cdk-lib/aws-iam';
 import { AwsEfsCreatorStackProps } from './AwsEfsCreatorStackProps';
+import { parsePerformanceMode, parseThroughputMode } from '../utils/efs-mode-parsing';
 
 /**
```

Comment:
**Suggestion:** Add a default case to the `parsePerformanceMode` and `parseThroughputMode` functions to handle undefined or empty strings, providing a more user-friendly error message.
> 🏡  **Medium Priority** 
 > [category: error handling, importance: 7]